cobigen-templates/templates-devon4j/src/main/templates/crud_java_server_app_complex/templates/java/${variables.rootPackage}/${variables.component}/logic/impl/usecase/UcManage${variables.entityName}Impl.java.ftl

```diff
@@ -13,6 +13,17 @@ import org.springframework.transaction.annotation.Transactional;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+<#assign compositeIdVar = false>
+<#list pojo.fields as field>
+	<#if field.type?starts_with("Composite")>
+		<#assign compositeIdVar = true>
+		<#assign compositeIdTypeVar = field.type>
+	</#if>
+</#list>
```

Comment:
missing refactoring

Reply:
Changes done and committed. Re-Review required.